.github/workflows/main.yml

```diff
@@ -52,10 +52,10 @@ jobs:
       run: brew install gcc@${GCC_V} || brew upgrade gcc@${GCC_V} || true
 
     - name: Configure with CMake
-      run: cmake -Wdev -S . -B . # We should build out of source but loadtxt needs fixing
+      run: cmake -Wdev -DCMAKE_BUILD_TYPE=Release -B build
```

Comment:
Don't you want to keep the `-S .` option? That's where the root directory is.

Reply:
`cmake -S` is a new feature for CMake 3.13. Since it seems the trend is to have CMake 3.14 as a minimum, it's fine to keep -S .